js_modules/dagster-ui/packages/ui-core/src/assets/AssetEvents.tsx

```diff
@@ -174,142 +233,71 @@ export const AssetEvents = ({
+                  description="No events found for the selected filters."
                 />
               </Box>
-            }
-          />
-          <MenuItem
-            shouldDismissPopover={false}
-            onClick={() => onToggle('observation')}
-            text={
-              <Box padding={{horizontal: 2}} flex={{direction: 'row', alignItems: 'center'}}>
-                <Checkbox
-                  size="small"
-                  checked={value.includes('observation')}
-                  onChange={() => {}}
-                  label="Observation"
-                />
+            );
+          }
+          return (
+            <>
+              <Box
+                style={{display: 'flex', flex: 1, minWidth: 200}}
+                flex={{direction: 'column'}}
+                background={Colors.backgroundLight()}
+              >
+                {loading && grouped.length === 0 ? (
+                  <Box flex={{alignItems: 'center', justifyContent: 'center'}} style={{flex: 1}}>
+                    <Spinner purpose="section" />
+                  </Box>
+                ) : (
+                  <AssetEventList
+                    xAxis={xAxis}
+                    groups={grouped}
+                    focused={focused}
+                    setFocused={onSetFocused}
+                    loading={loading}
+                    onLoadMore={fetchMore}
+                  />
+                )}
               </Box>
-            }
-          />
-        </Menu>
-      }
-    >
-      <Button
-        onClick={() => setShowMenu((current) => !current)}
-        icon={<Icon name="filter_alt" />}
-        rightIcon={<Icon name="expand_more" />}
-      >
-        Type ({value.length})
-      </Button>
-    </Popover>
+
+              <Box
+                flex={{direction: 'column'}}
+                style={{flex: 3, minWidth: 0, overflowY: 'auto'}}
+                border="left"
+              >
+                <ErrorBoundary region="event" resetErrorOnChange={[focused]}>
+                  {xAxis === 'partition' ? (
+                    focused ? (
+                      <AssetPartitionDetail
+                        group={focused}
+                        hasLineage={assetHasLineage}
+                        assetKey={assetKey}
+                        stepKey={assetNode ? stepKeyForAsset(assetNode) : undefined}
+                        latestRunForPartition={null}
+                        changedReasons={assetNode?.changedReasons}
+                      />
+                    ) : (
+                      <AssetPartitionDetailEmpty />
+                    )
+                  ) : focused?.latest ? (
+                    <AssetEventDetail assetKey={assetKey} event={focused.latest} />
+                  ) : (
+                    <AssetEventDetailEmpty />
+                  )}
+                </ErrorBoundary>
+              </Box>
+            </>
+          );
+        })()}
+      </Box>
+    </>
```

Comment:
This looks good but it'd be worth unmounting a code location and then opening this page to make sure that this renders reasonably for non-SDAs, a lot of the cruft in the old code was for that backcompat I think.

Reply:
Hmmm. Do you know any examples I could use from `dogfood-test-1` ?